components/src/core/ListItemTag/ListItemTag.tsx

```diff
@@ -13,22 +13,27 @@ export type ListItemTagProps = {
 
     /* The string label of the tag. */
     label: string;
-} & TypographyProps;
+};
 
 const useStyles = makeStyles((theme: Theme) =>
     createStyles({
         root: {
-            fontWeight: 'bold',
-            letterSpacing: 1,
             borderRadius: theme.spacing(0.25),
             padding: 0,
             paddingLeft: theme.spacing(0.5),
             paddingRight: theme.spacing(0.5),
-            lineHeight: 'inherit',
             overflow: 'hidden',
             backgroundColor: (props: ListItemTagProps): string => props.backgroundColor || theme.palette.primary.main,
             color: (props: ListItemTagProps): string => props.fontColor || theme.palette.primary.contrastText,
             cursor: (props: ListItemTagProps): string => (props.onClick ? 'pointer' : 'inherit'),
+            display: 'inline-block',
+        },
+        noVariant: {
+            fontWeight: 'bold',
```

Comment:
I am OK either way, but I recall you prefer fontWeight to be 700 instead of bold?